sympy/physics/mechanics/tests/test_kane.py

```diff
@@ -21,8 +21,10 @@ def test_one_dof():
     pa = Particle('pa', P, m)
     BL = [pa]
 
-    KM = KanesMethod(N, [q], [u], kd)
-    KM.kanes_equations(BL, FL)
+    KM = KanesMethod(N, [q], [u], kd, bodies=BL, forcelist=FL)
+    KM.kanes_equations()
```

Comment:
Both cases should still work, so don't remove the test, only add a new one.

Reply:
Okay